net/rsync/Portfile

```diff
@@ -9,7 +9,7 @@ categories          net
 license             GPL-3+
 installs_libs       no
 platforms           darwin freebsd sunos
-maintainers         nomaintainer
+maintainers         jimjag
```

Comment:
This would mean that you are a Macports committer. You probably mean to use your GitHub handle here, so this line should be:
`maintainers                @jimjag`

Reply:
Ideally you should also use the obfuscated email and add `openmaintainer` (unless there's a super strong reason to keep it under closed maintainership policy, but that's problematic when the maintainer doesn't have commit rights anyway).

Reply:
jimjag *is* a MacPorts committer; see https://trac.macports.org/wiki/jimjag. He used to maintain this port, but was removed last year when it was thought he was no longer participating in MacPorts; see https://trac.macports.org/ticket/52038. But yes, it is ideal to list both MacPorts handle and GitHub handle, like this: `maintainers {jimjag @jimjag}`

Reply:
Thanks for pointing this out. Usually he would have been listed as "member" rather than "contributor", meaning that he probably doesn't have commit rights.

Reply:
Yes, that's why I was confused. Why did he not simply commit this himself? And why not respond to the port abandoned ticket?